cmd/kusk/cmd/dashboard.go

```diff
@@ -54,23 +55,41 @@ The flags --envoyfleet.namespace and --envoyfleet.name can be used to change the
 	Expose dashboard on port 9090
 	`,
 	Run: func(cmd *cobra.Command, args []string) {
+		reportError := func(err error) {
+			if err != nil {
+				// Report error
+				miscInfo := map[string]interface{}{
+					"envoyfleet.namespace": dashboardEnvoyFleetNamespace,
+					"envoyfleet.name":      dashboardEnvoyFleetName,
+					"external-port":        dashboardEnvoyFleetExternalPort,
```

Comment:
this would collect public IP address of the user. Remove it

Reply:
Please mark resolved if you're happy with it.